t_fault_injection/test_fault_injection_base.py

```diff
@@ -172,7 +271,649 @@ def test_stress(self):
         )
         self.start_all_services(client=False)
 
-        self.setup_fail_function_test("tdb_alloc_blk", 10, -1, 0, 0)
+        TestFailFunctionBaseStress.setup_fail_function_test(func_name, 10, -1, 0, 0)
+        client = self.get_client(client_id)
+
+        self.oops_ignore = ["ERROR"]
+        client.start()
+        self.wait_while_busy(client, timeout=50)
+        client.stop()
+
+
+class TestSchedConfig(TestFailFunctionBase):
+    backends = [
+        {
+            "id": "deproxy_1",
+            "type": "deproxy",
+            "port": "8000",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        },
+        {
+            "id": "deproxy_2",
+            "type": "deproxy",
+            "port": "8001",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        },
+        {
+            "id": "deproxy_3",
+            "type": "deproxy",
+            "port": "8002",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        },
+        {
+            "id": "deproxy_4",
+            "type": "deproxy",
+            "port": "8003",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        },
+        {
+            "id": "deproxy_5",
+            "type": "deproxy",
+            "port": "8004",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        },
+    ]
+
+    base_tempesta_config = f"""
+listen 80;
+listen 443 proto=h2,https;
+tls_certificate {TEMPESTA_WORKDIR}/tempesta.crt;
+tls_certificate_key {TEMPESTA_WORKDIR}/tempesta.key;
+server {SERVER_IP}:8000 conns_n=10;
+"""
+
+    @marks.Parameterize.expand(
+        [
+            marks.Param(
+                name="tfw_sched_ratio_srvdesc_setup_srv_1",
+                func_name="tfw_sched_ratio_srvdesc_setup_srv",
+                extra_config=f"""
+server {SERVER_IP}:8001;
+server {SERVER_IP}:8002;
+server {SERVER_IP}:8003;
+server {SERVER_IP}:8004;
```

Comment:
Let's extract this server block to constant to have this:
```
DEF_SERVERS = f"""
server {SERVER_IP}:8001;
server {SERVER_IP}:8002;
server {SERVER_IP}:8003;
server {SERVER_IP}:8004;
"""
```
We can concat this constant with other parts of config.